config.go

```diff
@@ -37,7 +37,7 @@ func ParseSSHConfigFile(path string) (map[string]SSHClientOptions, error) {
 
 	// Read lines in reverse order and parse option for each Host section:
 	lines := strings.Split(string(content), "\n")
-	hostExpr := regexp.MustCompile("\\s*Host\\s*=?\\s*(.+)")
+	hostExpr := regexp.MustCompile("\\s*Host\\s*(=?)\\s*(.+)")
```

Comment:
I wonder if it's possible to just update this regexp as `"^\\s*Host\\b\\s*=?\\s*(.+)"` and we're done.

See https://play.golang.org/p/J8GdA1UOTv

Reply:
You right, that worked great. I pushed the changes. Thanks!